messages/jtd.js

```diff
@@ -175,6 +182,7 @@ module.exports = {
     cs: "musí být rovno jedné hodnotě z výčtu",
     de: "muss einem der vorgegebenen Werte entsprechen",
     es: "deber ser igual a uno de los valores predefinidos",
+    fi: "täytyy olla yhtä kuin jokin sallituista arvoista",
```

Comment:
probably worth making the message in "enum" above the same?

Reply:
Both "enum" properties should have identical translations now.